src/OpenTelemetry.Instrumentation.Dependencies/Implementation/HttpHandlerDiagnosticListener.cs

```diff
@@ -73,24 +73,24 @@ public override void OnStartActivity(Activity activity, object payload)
                 return;
             }
 
-            this.Tracer.StartActiveSpanFromActivity(Constants.HttpSpanPrefix + request.Method, activity, SpanKind.Client, out var span);
+            this.Tracer.StartActiveSpanFromActivity(HttpTagHelper.GetOperationNameForHttpMethod(request.Method), activity, SpanKind.Client, out var span);
```

Comment:
Suggestion: there are reasons to not include the path by default in the name but it is very useful on the root span, it will be good to have an option to add it to the span name.